src/components/products-section/ProductsSection.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Product from "../product/Product";
+import "./style.css";
+
+const ProductsSection = ({ title, products, category }) => {
+  return (
+    <>
+      <section className={`products-section products-section--${category}`}>
+        <h2>{title}</h2>
+        <div className="products">
+          {products.map((product, index) => {
+            const { name, desc, price, imgAlt, imgSrc } = product;
+            return (
+              <Product
+                key={index}
```

Comment:
bacha tady na ten key - používat index je poslední možnost, zkus použít vždy něco jinýho, unikátního. V tomhle případě by mohlo stačit "name" třeba
https://reactjs.org/docs/lists-and-keys.html#keys